src/index.ts

```diff
@@ -76,21 +76,28 @@ async function redirects(request: Request) {
   }
 
   if (url.subdomain === 'meet') {
-    if (url.pathname === '/') {
-      return Response.redirect('https://meet.google.com/vtk-ncrc-rdp')
-    } else if (url.pathname === '/dev') {
-      return Response.redirect('https://meet.google.com/rci-pize-jow')
-    } else if (url.pathname === '/einbindung') {
-      return Response.redirect('https://meet.google.com/qzv-ojgk-xqw')
-    } else if (url.pathname === '/begleitung') {
-      return Response.redirect('https://meet.google.com/kon-wdmt-yhb')
-    } else if (url.pathname === '/reviewing') {
-      return Response.redirect('https://meet.google.com/kon-wdmt-yhb')
-    } else if (url.pathname === '/labschool') {
-      return Response.redirect('https://meet.google.com/cvd-pame-zod')
-    } else {
-      return createNotFoundResponse()
-    }
+      switch(url.pathname){
+    case '/':
+        return Response.redirect('https://meet.google.com/vtk-ncrc-rdp')
+        break;
```

Comment:
`break` is not needed here (and in the other `case`s) since you already `return`. Furthermore: please run `yarn format` to reformat the code according to our coding style (this is the reason for the failing CI)